corehq/apps/ota/views.py

```diff
@@ -153,23 +153,24 @@ def get_restore_params(request):
     Given a request, get the relevant restore parameters out with sensible defaults
     """
     # not a view just a view util
+    query_dict = request.GET or request.POST
```

Comment:
how come this was necessary?

Reply:
Currently the HMAC validation uses the request body to generate the hash. We could change that to using the query params for a GET request.

Side note - I also want to mandate that hmac validated requests include a `timestamp` request param to make sure that replay attacks harder.